request_logging/middleware.py

```diff
@@ -139,7 +144,13 @@ def _should_log_route(self, request):
         elif hasattr(view, 'view_class'):
             # This is for django class-based views
             func = getattr(view.view_class, method, None)
+
         no_logging = getattr(func, NO_LOGGING_ATTR, None)
+        if self.no_logging_get_response:
```

Comment:
You can collapse this into one `if` statement. It would be good to add a test or two to prove expected behavior.